modules/tinymce/src/plugins/link/test/ts/browser/AllowUnsafeLinkTargetTest.ts

```diff
@@ -1,78 +1,79 @@
-import { Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
-import LinkPlugin from 'tinymce/plugins/link/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import { describe, it, before, after } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
```

Comment:
This reminds me that I need to discuss with the team about enabling "grouping" in imports. James and I had discussed it and had been following that while migrating.

For clarification, it's just about grouping external/internal imports, so for here there'd be a new line above this. Anyways, not something to worry about for now.